examples/lwt/dune

```diff
@@ -1,9 +1,4 @@
 (executables
- (names lwt_get lwt_get_pipelined echo_server multipart_upload_server)
- (modules lwt_get lwt_get_pipelined echo_server multipart_upload_server)
- (libraries piaf logs logs.fmt fmt.tty multipart))
-
-(executable
- (name echo_server_upgrade)
- (modules echo_server_upgrade)
- (libraries piaf digestif.c websocketaf))
+ (names lwt_get lwt_get_pipelined multipart_upload_server)
```

Comment:
This is a no-go. Examples are useful and serve a purpose.

if you don't want to build the examples, or otherwise things that are not part of the package, just run `dune build @install` rather than `dune build` (which [defaults](https://dune.readthedocs.io/en/stable/usage.html#default-alias) to `@all`)

Reply:
I'll revert his one